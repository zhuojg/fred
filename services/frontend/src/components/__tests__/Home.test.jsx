import React from 'react'
import { cleanup, wait } from '@testing-library/react'

import Home from '../Home'

afterEach(cleanup)

describe('renders', () => {
  test('snapshot', () => {
    const { asFragment } = renderWithRouter(<Home />)
    expect(asFragment()).toMatchSnapshot()
  })
})
