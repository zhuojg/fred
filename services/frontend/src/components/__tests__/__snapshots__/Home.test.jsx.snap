// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="home_welcome_wrap"
    >
      <div
        class="home_welcome"
      >
        <div
          class="ant-row ant-row-center"
        >
          <div
            class="ant-col ant-col-12"
          >
            <div
              class="home_welcome_text"
            >
              <div
                class="home_welcome_text_title"
              >
                Meet FRED: Flask + REact + Docker
              </div>
              <div
                class="home_welcome_text_desc"
              >
                An End-to-End Boilerplate for Full Stack Development
              </div>
              <div
                class="home_welcome_text_button_wrap"
              >
                <button
                  class="ant-btn ant-btn-primary ant-btn-round ant-btn-lg"
                  style="margin-right: 8px;"
                  type="button"
                >
                  <span>
                    What Fred uses?
                  </span>
                </button>
                <button
                  class="ant-btn ant-btn-round ant-btn-lg"
                  type="button"
                >
                  <span>
                    Where is Fred?
                  </span>
                </button>
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-12"
          >
            <div
              class="home_welcome_img"
            >
              <a
                href="https://undraw.co/"
              >
                <img
                  alt="fred at work"
                  src="/img/illustrations/designer.svg"
                />
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="use_wrap"
      id="what-fred-uses"
    >
      <div
        class="use_title_wrap"
      >
        <div
          class="use_title"
        >
          What Fred Uses
        </div>
        <div
          class="use_subtitle"
        >
          to make you happy
        </div>
        <div
          class="use_divider"
        />
      </div>
      <div
        class="icon_wrap"
      >
        <div
          class="ant-row ant-row-space-around"
          style="margin: -8px -8px 8px -8px;"
        >
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="flask-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Flask
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="logo-react"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                React
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="cube-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Docker
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="server-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Postgres
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-row ant-row-space-around"
          style="margin: -8px -8px 8px -8px;"
        >
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="layers-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                SQLAlchemy
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="logo-css3"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Bulma
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="git-compare-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                CircleCI
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="rocket-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Heroku
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-row ant-row-space-around"
          style="margin: -8px -8px 8px -8px;"
        >
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="logo-nodejs"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Nodejs
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="logo-npm"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                NPM
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="flashlight-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Jest
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="reader-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Formik
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-row ant-row-space-around"
          style="margin: -8px -8px 8px -8px;"
        >
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="logo-python"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Python
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="hammer-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                PyTest
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="key-outline"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                JWT
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-3"
            style="padding: 8px 8px 8px 8px;"
          >
            <div
              class="icon_item_wrap"
            >
              <div
                class="icon_item"
              >
                <span
                  class="anticon"
                  role="img"
                >
                  <ion-icon
                    name="logo-github"
                    size="large"
                  />
                </span>
              </div>
              <div
                class="icon_title"
              >
                Github
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="quotes_wrap"
    >
      <div
        class="quotes_mask"
      />
      <div
        class="quotes_content"
      >
        <div
          class="quotes_title"
        >
          Quotes of the Day
        </div>
        <div
          class="quotes_subtitle"
        >
          <a
            href="http://source.unsplash.com"
          >
            Random Background Image from Unsplash
          </a>
        </div>
        <div
          class="quotes_cards"
        >
          <div
            class="ant-row ant-row-center"
            style="margin-left: -12px; margin-right: -12px;"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
